### ISSUE TITLE
Show and check network identity

### ISSUE THREAD

**Issue by jsign** [textileio/powergate#467]
Include in the starting log output which network is being targeted, and if possible check if the connected Lotus node is running in the same network.

**Comment by ArseniiPetrovich** [comment]
Currently we are working on Filecoin cloud images solution which will provide the end users with possibility to run the Filecoin nodes that will have included ledger state. In this images we also have the Powergate application included to provide users with better experience. We are building images for both test and interop networks, but as I know from my blockchain experience the number of networks will grow and change all the time, and some of the users might want to run their own networks as well.
In our solution we rely on branches and tags allowing to include and exclude networks from daily build fast. However it seems that Powergate is working with the latest network version only, making impossible for end users to connect to different networks at the same time. As a workaround I'm skipping the Powergate status check for now, but it would be really nice and useful to have an kind of cli argument that will allow to override bootstrap nodes. It could be either `--bootstrap-nodes <list>` which accepts a list of endpoints; or maybe `--network=<name>` that will use a one of the predefined bootstrap nodes list; or both.

**Comment by jsign** [comment]
@ArseniiPetrovich , I have a PR in progress. In the PR description I describe the solution that I've found.
TL;DR: You might ignore this problem and let `powd` figure out the right network for you, leveraging information provided by the Lotus node is connecting to. So you only have to use the right Lotus node configuration, and just run `powd` as usual.

Let me know if make sense for you.

**Comment by ArseniiPetrovich** [comment]
It would be great, looking forward for this PR to be merged! @jsign
